src/js/transactions.js

```
@@ -0,0 +1,18 @@
+import {createApp} from 'vue';
+import {createPinia} from 'pinia';
+
+import Transactions from './components/Transactions.vue';
+import '../css/admin/transactions.css';
+
+if (process.env.NODE_ENV === 'development') {
+    app.config.devtools = true;
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)